tools/virt-host-validate-ch.c

```diff
@@ -0,0 +1,129 @@
+#include <config.h>
+
+#include "virt-host-validate-ch.h"
+#include "virt-host-validate-common.h"
+#include "virarch.h"
+#include "virbitmap.h"
+
+int validateMinimumChVersion(char* ver_string)
+{
+	// expected ver_string example: "cloud-hypversor v0.8.1-<hash>"
+    int min_major_version = 0;
+    int min_minor_version = 9;
+    char *ptr = strtok(ver_string, " ");
+    char *last_tok = NULL;
+    while(ptr != NULL)
+	{
+        last_tok = ptr;
+		ptr = strtok(NULL, " ");
```

Comment:
There seems to be a few coding style issues with that PR. Have you run `ninja -C build test` against it?

Reply:
Addressed all test defects in `ninja -C build test`